.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,6 +1,6 @@
 ---
 name: 🐛 Bug report
-about: Create a report to help us improve
+about: Create a bug report, only if you can reproduce on master branch.
```

Comment:
```suggestion
about: Create a bug report (only if it's reproducible on the master branch)
```

Reply:
@gundalow What do you think? Which one fits better with Cambridge English? ;)

Reply:
@awcrosby What do you think? Comma or parentheses for the descriptions used on https://github.com/ansible/ansible-lint/issues/new/choose ?